NodeJS/src/SubscriptionClient.ts

```diff
@@ -0,0 +1,79 @@
+import {Session} from "./session";
+import {Subscription} from "./Subscription";
+
+const VERSION = '1.0';
+const SUBSCRIPTION_PATH = '/spokenAbc/subscriptions/v' + VERSION + '/subscriptions';
+const SLASH = '/';
+const SUB_ACCOUNT_KEY = 'subAccountAppId=';
+const QUESTION_MARK = '?';
+
+class SubscriptionClient {
+    session:Session;
+
+    constructor(session: any) {
+        this.session = session
+    }
+
+    async createSubscription(createSubscriptionRequest:Subscription) {
+        await this.session.login();
+
+        let subAccountAppId = await this.getSubAccountAppId();
+        createSubscriptionRequest.subAccountAppId = subAccountAppId;
+        let returnedSubscription = await this.session.post(
+            SUBSCRIPTION_PATH + QUESTION_MARK + SUB_ACCOUNT_KEY + subAccountAppId,
+            createSubscriptionRequest)
+            .then(result => result.data);
+
+        return returnedSubscription
+    }
+
+    async getAllSubscriptions() {
+        await this.session.login();
```

Comment:
Wondering is there a logout()?  What is the behavior of multiple login() called on the session object.